webpack.config.dev.js

```diff
@@ -1,6 +1,4 @@
 const path = require('path');
-const webpack = require('webpack');
-const pkg = require('./package.json');
```

Comment:
Lets remove `disableHostCheck: true,` from the devServer options. This is not recommended configuration

Reply:
Also, the only different between this config and the non-dev configuration should be the `mode`, `output.path`, `devServer` and `devTool` config. IMO we should combine these files.